### DISCUSSION TITLE
Welcome to wtfpython-ko Discussions!

### DISCUSSION THREAD

**Discussion by buttercrab** [buttercrab/wtfpython-ko#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by uhhyunjoo** [comment]
안녕하세요! 원문 레포를 알게 되어 찾아보다가 이 레포를 발견하게 된 학부생 입니다.
Translate Progress에서 Class attributes and instance attributes이 아직 번역되지 않은 걸 확인했는데
혹시 지금 그 부분을 하고 계시는 분이 없다면 저도 참여해서 번역할 수 있을까요?

**Reply by buttercrab** [reply]
넵 감사합니다.

**Reply by buttercrab** [reply]
[Progress](https://github.com/buttercrab/wtfpython-ko/discussions/31#discussion-52588)를 업데이트 했습니다.
